src/ocean/util/serialize/contiguous/package_.d

```diff
@@ -734,10 +734,10 @@ unittest
 {
     static struct CS
     {
-        cstring s;
+        mstring s;
```

Comment:
Ok, we talked with @mathias-lang-sociomantic, looks like it was needed for one of work-in-progress ports and doesn't affect any merged code. We'll see if that code can be adjusted in a fashion simlar to swarm, for now this adjustments should be OK to merge.